### PR TITLE
Misc typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3`

### CODE REVIEW

File: doc/sphinxext/numpydoc/tests/test_docscrape.py [hunk 1]
```diff
@@ -340,7 +340,7 @@ def test_sphinx_str():
     **out** : ndarray
 
         The drawn samples, arranged according to `shape`.  If the
-        shape given is (m,n,...), then the shape of `out` is is
+        shape given is (m,n,...), then the shape of `out` is
```
Comment (luzpaz) [inline doc/sphinxext/numpydoc/tests/test_docscrape.py]: These changes are in upstream numpy already.

File: asv_bench/benchmarks/rolling.py [hunk 2]
```diff
@@ -30,12 +30,12 @@ class Quantile(object):
...
 
-    def setup(self, contructor, window, dtype, percentile):
+    def setup(self, constructor, window, dtype, percentile):
         N = 10**5
         arr = np.random.random(N).astype(dtype)
-        self.roll = getattr(pd, contructor)(arr).rolling(window)
+        self.roll = getattr(pd, constructor)(arr).rolling(window)
 
-    def time_quantile(self, contructor, window, dtype, percentile):
+    def time_quantile(self, constructor, window, dtype, percentile):
```
Comment (luzpaz) [inline asv_bench/benchmarks/rolling.py]: please confirm source changes here. I didn't think changing them would impact anything critical.

Comment (jreback) [reply asv_bench/benchmarks/rolling.py]: this is ok

**Comment by jreback** [comment]
thanks!

**Pull request merged**